styles.css

```diff
@@ -0,0 +1,598 @@
+  background-color: white;
+  color: grey;
+}
+
+
+.card {
+  display: flex;
+  width: 250px;
+  align-self: center;
+  margin-top: 10px;
+  background-color: #F2F2F3;
+  color: grey;
+  filter: drop-shadow(1px 1px 1px gray);
+
+
+}
+
+.card-header {
+  align-self: flex-start;
+  height: 100px;
+  width: 250px;
+  /* background-color: #0AF5E8; */
+  border-color: red;
+  display: flex;
+  flex-direction: column;
+}
+
+.card-url {
+  align-self: center;
+  background-color: white;
+  width: 200px;
+  height: 50px;
+  color: grey;
+  font-size: 12px;
+  justify-content: flex-start;
+  text-decoration: none;
+
+}
+
+.label-container {
+  height: 10vh;
+  width: 1030px;
+  align-items: center;
+  align-self: center;
+}
+
+.nav-right {
+  background-color: green;
+  height: 10vh;
+  width: 250px;
+  display: flex;
+  flex-direction: row-reverse;
+  background-color: inherit;
+}
+
+.nav-text {
+  color: white;
+  margin: 0px;
+  width: 150px;
+  font-size: 12px;
+  margin: 5px;
+  background-color: inherit;
+  align-self: flex-end;
+  padding-bottom: 10px;
+}
+
+.nav-text-all-time {
+  color: white;
+  margin: 0px;
+  width: 150px;
+  font-size: 12px;
+  margin: 5px;
+  background-color: inherit;
+  color: black;
+  align-self: flex-end;
+  padding-bottom: 10px;
+
+
+
+}
+
+.streamers {
+  font-size: 36px;
+  padding-top: 20px;
+  color: gray;
+  flex-direction: column;
+}
+
+
+.user-menu {
+  color: white;
+  margin: 0px;
+  width: 200px;
+  flex-direction: column;
+  font-size: 12px;
+  margin: 5px;
+  background-color: inherit;
+  flex-wrap: wrap;
+
+}
+
+.url {
+  padding-left: 3px;
+  padding-top: 3px;
+}
+
+.ninja {
+  background-image: url('./assets/header2_ninja.png');
+}
+
+.ludwig {
+  background-image: url('./assets/header2_ludwig.png');
+}
+
+.ranboolive {
+  background-image: url('./assets/header2_ranboolive.png')
+}
+
+.shroud {
+  background-image: url('./assets/header2_shroud.png');
+}
+
+.summit {
+  background-image: url('./assets/header2_summit.png');
+}
+
+.xqc {
+  background-image: url('./assets/header2_xqc.png');
+}
+
+.trymacs {
+  background-image: url('./assets/header2_trymacs.png');
+}
+
+.nickmercs {
+  background-image: url('./assets/header2_nickmercs.png');
+}
+
+.tfue {
+  background-image: url('./assets/header2_tfue.png');
+  outline: none;
+  border: none;
+}
+
+.hasanabi {
+  background-image: url('./assets/header2_hasanabi.png');
+  outline: none;
+  border: none;
+}
+
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+@media screen and (max-width: 1040px) {
+
+html,
+body
+main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  font-family: 'Dela Gothic One', cursive;
+  font-family: 'Lato', sans-serif;
+  overflow: scroll;
+
+}
+
+
+article {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  border: none;
+
+
+}
+
+button {
+  outline: none;
+  border: none;
+
+}
+
+div {
+
+}
+
+header {
+  background-color: #0AF5E8;
+  height: 15vh;
```

Comment:
I like the use of `vh` here, I wonder if there are other areas of the code where `vh` and `vw` might give you more flexibility than pixels as well!